Simplenote/StoreManager.swift

```diff
@@ -91,11 +95,26 @@ class StoreManager {
         Task {
             do {
                 try await purchase(product: product)
+
+                Task { @MainActor in
+                    /// Note: This must be invoked in the main thread, otherwise, warnings get triggered!
+                    SPTracker.trackSustainerPurchaseCompleted(storeProduct: storeProduct)
```

Comment:
Then can we annotate `SPTracker` with `@MainActor`? That way, the compiler will enforce it for us!

Reply:
Done, thank you BJ!!